packages/messenger/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Migrate `Messenger` class from `@metamask/base-controller` package
```

Comment:
Nit: I believe the changelog is failing because if pr is not included
```suggestion
- Migrate `Messenger` class from `@metamask/base-controller` package ([#6127](https://github.com/MetaMask/core/pull/6127))

Reply:
Ah darn, thanks, good catch!